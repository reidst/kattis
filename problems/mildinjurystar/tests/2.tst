2 2 3
0 0 S 0
0 0 E 1
0 1 S 1
