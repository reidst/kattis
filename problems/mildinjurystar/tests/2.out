March the 30th be with you!
